src/main/java/com/example/demo/services/LeaseManagementDropBoxImpl.java

```diff
@@ -38,22 +43,26 @@ public class LeaseManagementDropBoxImpl implements LeaseManagement {
 
     @Autowired
     public LeaseManagementDropBoxImpl(UserRepository userRepository, LeaseRepository leaseRepository, ApartmentRepository apartmentRepository, TenantRepository tenantRepository) {
-        ApiClient apiClient = com.dropbox.sign.Configuration.getDefaultApiClient().setApiKey(dropBoxSignToken);
-        this.signatureRequestApi = new SignatureRequestApi(apiClient);
         this.userRespository = userRepository;
         this.leaseRepository = leaseRepository;
         this.apartmentRepository = apartmentRepository;
         this.tenantRepository = tenantRepository;
     }
 
+    @PostConstruct
+    private void initApiClient() {
+        ApiClient apiClient = com.dropbox.sign.Configuration.getDefaultApiClient().setApiKey(dropBoxSignToken);
+        this.signatureRequestApi = new SignatureRequestApi(apiClient);
+    }
+
 
     @Transactional(rollbackFor = Exception.class)
     public SignatureRequestGetResponse createLeaseSignatureRequest(LeaseSignRequestDTO leaseSignRequestDTO) throws ApiException {
         SignatureRequestGetResponse response;
         Optional<User> userRecord = userRespository.findByEmailIgnoreCase(leaseSignRequestDTO.getSignerEmails().getFirst());
         //User user = userRecord.orElseThrow(() -> new EmptyResultDataAccessException("user not found", 1));
         //todo this was set so we can test without setting up user account each time each time
-        User user = userRecord.orElse(userRespository.save(User.builder().email(leaseSignRequestDTO.getSignerEmails().getFirst()).name("test user created").password("test").username(leaseSignRequestDTO.getSignerUserName() != null ? leaseSignRequestDTO.getSignerUserName() : "default").build()));
+        User user = userRecord.orElse(userRespository.save(User.builder().email(leaseSignRequestDTO.getSignerEmails().getFirst()).name("test user created").password("test").username(leaseSignRequestDTO.getSignerUserName() != null ? leaseSignRequestDTO.getSignerUserName() : "test").build()));
```

Comment:
Do you still need this? You can take a test account from the database. I can show you tonight. You can also add test accounts yourself.